_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -796,7 +796,7 @@ def convert_zip_to_hrr(self, data, zip_col="zip", hrr_col="hrr"):
         return data
 
     def convert_zip_to_msa(
-        self, data, zip_col="zip", msa_col="msa", date_col="date", count_cols=None
+        self, data, zip_col="zip", date_col="date", count_cols=None
```

Comment:
What's going on here?

Reply:
that variable is unused in that function.

Reply:
Do we know if any indicators call it with that keyword?

Reply:
Didn't find any uses when I searched, only saw use of the `zip_to_msa` function below it, which calls `convert_zip_to_msa`.